airflow/models.py

```diff
@@ -3755,7 +3766,8 @@ def clear_dags(
             confirm_prompt=False,
             include_subdags=True,
             reset_dag_runs=True,
-            dry_run=False):
+            dry_run=False,
+            only_include_backfill_dagruns=False):
```

Comment:
nit: break line + trailing comma. For the reason that if this had been done before you wouldn't be on git blame for the line above...

Reply:
updated